CHANGELOG.md

```diff
@@ -2,6 +2,20 @@
 
 # Changelog
 
+## v1.15.1 (2020-07-27)
+    Mask over the /sys/fs/selinux in mask branch
+    chroot: do not use setgroups if it is blocked
+    chroot, run: not fail on bind mounts from /sys
+    Allow "readonly" as alias to "ro" in mount options
+    Add VFS additional image store to container
+    Ignore OS X specific consistency mount option
+    vendor golang.org/x/text@v0.3.3
+    Cirrus: Fix missing htpasswd in registry img
+    Switch scripts to use containers.conf
+    Merge pull request #2471 from vrothberg/1.15-update-x/text
+    Make imagebuildah.BuildOptions.Architecture/OS optional
+    Merge pull request #2483 from nalind/release-1.15-arch
```

Comment:
This is the merge commit log for the previous line.  We don't need to mention it.